tests/testing_common.py

```diff
@@ -352,6 +352,21 @@ def _test_generate_half_prec(self, model_id, config_cls, config_kwargs):
             # check if `generate` raises an error if no positional arguments are passed
             _ = model.generate(input_ids, attention_mask=attention_mask)
 
+    def _test_prefix_tuning_half_prec_conversion(self, model_id, config_cls, config_kwargs):
```

Comment:
You need to add that in `test_decoder_models` and `test_encoder_decoder_models` as well, check out for example: https://github.com/huggingface/peft/blob/c46d76ae3a323fa208bd777b9eb60a383e094d49/tests/test_decoder_models.py#L97